cypress/support/pages/product.pageObject.js

```diff
@@ -0,0 +1,17 @@
+/// <reference types="Cypress" />
+
+import PageObject from '../PageObject';
+
+class ProductPageObject extends PageObject {
+  selectProduct(productName) {
+    cy.contains('.hrefch', productName)
```

Comment:
The use of 'hrefch' class seems incorrect. It might not select the desired element if 'hrefch' is not a valid class. Make sure the class name you are using to select the product is correct.

Reply:
Try `.card-block .card-title` instead

Reply:
done